tests/Control/SMIMEMailerTest.php

```diff
@@ -0,0 +1,243 @@
+<?php
+
+namespace SilverStripe\SMIME\Tests\Control;
+
+use Exception;
+use PHPUnit\Framework\MockObject\MockObject;
+use SilverStripe\Control\Email\Email;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\SMIME\Control\SMIMEMailer;
+use Swift_ByteStream_FileByteStream;
+
+class SMIMEMailerTest extends SapphireTest
+{
+
+    protected $usesDatabase = false; // phpcs:ignore
+
+    public function testEncryptionOnly()
```

Comment:
Might be worth having some docblocks on these too, just to explain what you are checking.

```suggestion
    /**
     * Check the email content is not sent in clear text and the correct content type is set
     *
     * @return void
     */
    public function testEncryptionOnly(): void
```